packages/react-components/react-text/src/components/Caption2/Caption2.tsx

```diff
@@ -0,0 +1,30 @@
+import type { FunctionComponent } from 'react';
+import { makeStyles } from '@griffel/react';
+import { createWrapper, TextWrapperProps } from '../wrapper';
+import { SlotClassNames } from '@fluentui/react-utilities';
+import { TextSlots } from '../Text/Text.types';
+import { typographyStyles } from '@fluentui/react-theme';
+
+/**
+ * @deprecated Use `caption2ClassNames.root` instead.
+ */
+export const caption2ClassName = 'fui-Caption2';
```

Comment:
Should we ship the deprecated `className` for new components too? Conformance is currently enforcing it; I assume for consistency reasons.

Reply:
if we are introducing breaking change like this IMO this is good opportunity to get rid of it. what's the issue with conformance ?

Reply:
I must've been sleepwalking, there is no error anymore when removing it 😶 Pushing the change now

Reply:
![image](https://user-images.githubusercontent.com/39736248/168117405-4b252d6e-b4f1-48fc-872f-61cbf8451d44.png)

https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=248744&view=logs&j=258ec178-2d8b-5611-7b9b-60c5c95dae55&t=8250ef03-1890-59ab-a6a8-5d0b4c2baa1b&l=1592

Reply:
You should remove `className`. We have another [issue to remove all the deprecated instances](https://github.com/microsoft/fluentui/issues/22132).

For conformance, you can disable the tests. Here's an example of how to do that: https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-spinbutton/src/components/SpinButton/SpinButton.test.tsx#L17

Reply:
thanks @spmonahan !